{"level":"error","ts":1630590739.485743,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithError: ERROR: duplicate key value violates unique constraint \"users_email_key\" (SQLSTATE 23505)","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":400,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:27\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:71\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"info","ts":1630590861.7144213,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"error","ts":1630591502.6603389,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithError: ERROR: duplicate key value violates unique constraint \"users_email_key\" (SQLSTATE 23505)","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":400,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:27\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:71\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630591506.058264,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithError: ERROR: duplicate key value violates unique constraint \"users_email_key\" (SQLSTATE 23505)","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":400,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:27\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:71\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630591509.9048824,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:65\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630591512.6487093,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithError: error: conflicting key 'required' rule 'Unknown' with text '{0} is a required field' for locale 'en', value being ignored","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":400,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:27\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:71\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"info","ts":1630591758.653093,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"error","ts":1630591777.2383244,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:65\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630591783.0239902,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Password' Error:Field validation for 'Password' failed on the 'gte' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:65\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630591786.3076618,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Password' Error:Field validation for 'Password' failed on the 'gte' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:65\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630591792.1614673,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithError: ERROR: duplicate key value violates unique constraint \"users_email_key\" (SQLSTATE 23505)","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":400,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:27\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:71\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"info","ts":1630593594.1054864,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"error","ts":1630593656.490676,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithError: ERROR: duplicate key value violates unique constraint \"users_email_key\" (SQLSTATE 23505)","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":400,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:27\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630593661.6696572,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:94\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630593666.400643,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag\nKey: 'User.Password' Error:Field validation for 'Password' failed on the 'gte' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:94\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"info","ts":1630594673.716998,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"info","ts":1630600793.0734444,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"info","ts":1630600995.4037638,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"info","ts":1630601193.7931495,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"info","ts":1630601208.9575188,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"info","ts":1630601251.5852199,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"info","ts":1630601478.1077683,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"info","ts":1630601503.036416,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"error","ts":1630601511.226942,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithError: no rows in result set","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users/:id","request_method":"DELETE","response_code":400,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:27\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).archive\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:153\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630601513.965857,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithError: no rows in result set","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users/:id","request_method":"DELETE","response_code":400,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:27\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).archive\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:153\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630601526.3958955,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithError: Key: 'archiveUserRequest.Password' Error:Field validation for 'Password' failed on the 'required' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users/:id","request_method":"DELETE","response_code":400,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:27\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).archive\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:133\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"info","ts":1630601749.1218016,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"error","ts":1630601756.0005574,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithError: Key: 'archiveUserRequest.Password' Error:Field validation for 'Password' failed on the 'required' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users/:id","request_method":"DELETE","response_code":400,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:27\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).archive\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:133\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630601770.0952425,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithError: no rows in result set","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users/:id","request_method":"DELETE","response_code":400,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:27\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).archive\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:153\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"info","ts":1630601889.1636379,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"error","ts":1630601900.2441826,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithError: no rows in result set","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users/:id","request_method":"DELETE","response_code":400,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:27\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).archive\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:153\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"info","ts":1630601918.2346687,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"info","ts":1630602033.0648863,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"error","ts":1630602039.1607869,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithError: user with given id not found.","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users/:id","request_method":"DELETE","response_code":400,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:27\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).archive\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:153\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"info","ts":1630602202.1143544,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"error","ts":1630602212.9103131,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithError: user with id 1 not found.","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users/:id","request_method":"DELETE","response_code":400,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:27\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).archive\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:153\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"info","ts":1630602226.2302856,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"error","ts":1630602377.917893,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithError: ERROR: duplicate key value violates unique constraint \"users_email_key\" (SQLSTATE 23505)","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":400,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:27\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:109\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630602384.3470514,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:104\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630602511.706096,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithError: incorrect password","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users/:id","request_method":"DELETE","response_code":400,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:27\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).archive\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:153\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"info","ts":1630602668.6446605,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"error","ts":1630648088.3867571,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:108\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630648092.2739155,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithError: ERROR: duplicate key value violates unique constraint \"users_email_key\" (SQLSTATE 23505)","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":400,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:27\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:113\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630648100.9631226,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:108\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"info","ts":1630648457.4432015,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"error","ts":1630648461.5245929,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:109\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630648463.770462,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":400,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:27\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:93\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630648467.8268747,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":400,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:27\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:93\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630648469.9287472,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":400,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:27\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:93\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630648471.5375824,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":400,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:27\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:93\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"info","ts":1630648725.4431398,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"error","ts":1630648735.6316736,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:113\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630648737.7423224,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:113\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630648739.4222426,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:113\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630648740.9472077,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:113\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"info","ts":1630649339.7332761,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"error","ts":1630649347.1307845,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:95\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630649349.3239565,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":400,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:27\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:79\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630649352.4101393,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":400,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:27\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:79\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"info","ts":1630649506.3699937,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"info","ts":1630649636.398004,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"info","ts":1630649842.0047677,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"info","ts":1630649905.1044655,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"info","ts":1630649948.8293445,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"info","ts":1630650030.8580558,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"info","ts":1630650096.0866146,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"error","ts":1630650250.5016816,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:93\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630650252.7302797,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:93\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630672223.1826882,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:97\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630672239.4163094,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:97\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630672241.521502,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:97\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630672242.528281,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:97\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"info","ts":1630672714.8353534,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"error","ts":1630672720.1486635,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:100\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630672722.7874796,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:100\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630672724.5161734,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:100\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"info","ts":1630673103.7142203,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"error","ts":1630673112.1104038,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:108\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630673114.8490913,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:108\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630673117.0088823,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:108\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"info","ts":1630674140.0637066,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"error","ts":1630674146.230215,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:70\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630674148.5385418,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:70\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630674150.0808623,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:70\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630674151.0353513,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:70\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630674159.702691,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Password' Error:Field validation for 'Password' failed on the 'gte' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:70\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"info","ts":1630674296.7373803,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"error","ts":1630674311.0260139,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Password' Error:Field validation for 'Password' failed on the 'gte' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:68\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630674313.3460648,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Password' Error:Field validation for 'Password' failed on the 'gte' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:68\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630674314.8815186,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Password' Error:Field validation for 'Password' failed on the 'gte' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:68\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630674316.263173,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Password' Error:Field validation for 'Password' failed on the 'gte' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:68\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630674318.4208407,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Password' Error:Field validation for 'Password' failed on the 'gte' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:68\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630674322.5740457,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag\nKey: 'User.Password' Error:Field validation for 'Password' failed on the 'gte' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:68\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630674324.843941,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag\nKey: 'User.Password' Error:Field validation for 'Password' failed on the 'gte' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:68\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630674328.3790183,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Password' Error:Field validation for 'Password' failed on the 'gte' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:68\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630674329.9969826,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Password' Error:Field validation for 'Password' failed on the 'gte' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:68\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630674333.8449547,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithError: ERROR: duplicate key value violates unique constraint \"users_email_key\" (SQLSTATE 23505)","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":400,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:27\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:73\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630674339.1283011,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithError: ERROR: duplicate key value violates unique constraint \"users_email_key\" (SQLSTATE 23505)","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":400,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:27\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:73\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"info","ts":1630674491.931491,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"error","ts":1630674499.5755155,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithError: ERROR: duplicate key value violates unique constraint \"users_email_key\" (SQLSTATE 23505)","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":400,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:27\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:73\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630674503.9600263,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:68\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630674505.5813224,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:68\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630674510.0086594,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag\nKey: 'User.Password' Error:Field validation for 'Password' failed on the 'gte' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:68\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630674512.403691,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag\nKey: 'User.Password' Error:Field validation for 'Password' failed on the 'gte' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:68\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"info","ts":1630674669.4487898,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"error","ts":1630674675.8931677,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag\nKey: 'User.Password' Error:Field validation for 'Password' failed on the 'gte' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:64\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630674677.8803508,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithError: error: conflicting key 'required' rule 'Unknown' with text '{0} is a required field' for locale 'en', value being ignored","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":400,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:27\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:60\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"info","ts":1630674740.881455,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"error","ts":1630674744.6241748,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag\nKey: 'User.Password' Error:Field validation for 'Password' failed on the 'gte' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:64\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630674746.3573725,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag\nKey: 'User.Password' Error:Field validation for 'Password' failed on the 'gte' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:64\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630674747.6329668,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag\nKey: 'User.Password' Error:Field validation for 'Password' failed on the 'gte' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:64\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630674748.387229,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag\nKey: 'User.Password' Error:Field validation for 'Password' failed on the 'gte' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:64\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"info","ts":1630674836.458361,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"error","ts":1630674839.9287174,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag\nKey: 'User.Password' Error:Field validation for 'Password' failed on the 'gte' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:64\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630674841.7439165,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'email' tag\nKey: 'User.Password' Error:Field validation for 'Password' failed on the 'gte' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users","request_method":"POST","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).create\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:64\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"info","ts":1630674846.9822922,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"error","ts":1630674920.699457,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'required' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users/:id","request_method":"DELETE","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).archive\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:119\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630674953.9049482,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithValidationError: Key: 'User.Email' Error:Field validation for 'Email' failed on the 'required' tag","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users/:id","request_method":"DELETE","response_code":422,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithValidationError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:32\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).archive\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:119\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"info","ts":1630674955.1432946,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"error","ts":1630674961.2566981,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithError: incorrect password","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users/:id","request_method":"DELETE","response_code":400,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:27\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).archive\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:111\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"error","ts":1630675002.1749961,"caller":"logger/logger.go:62","msg":"http - v1 - abortWithError: incorrect password","app-name":"go-service-template","app-version":"1.0.0","path":"/api/v1/users/:id","request_method":"DELETE","response_code":400,"stacktrace":"github.com/ysomad/go-auth-service/pkg/logger.(*logger).error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:62\ngithub.com/ysomad/go-auth-service/pkg/logger.Error\n\t/home/alex/code/my/go-auth-service/pkg/logger/logger.go:31\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.logError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:17\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.abortWithError\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/error.go:27\ngithub.com/ysomad/go-auth-service/internal/delivery/http/v1.(*userRoutes).archive\n\t/home/alex/code/my/go-auth-service/internal/delivery/http/v1/user.go:111\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.CustomRecoveryWithWriter.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/recovery.go:99\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.LoggerWithConfig.func1\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/logger.go:241\ngithub.com/gin-gonic/gin.(*Context).Next\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP\n\t/home/alex/go/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445\nnet/http.serverHandler.ServeHTTP\n\t/snap/go/8172/src/net/http/server.go:2878\nnet/http.(*conn).serve\n\t/snap/go/8172/src/net/http/server.go:1929"}
{"level":"info","ts":1630675049.4315946,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"info","ts":1630675094.2725127,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"info","ts":1630675229.5178285,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"info","ts":1630675274.6903608,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
{"level":"info","ts":1630675340.7666702,"caller":"logger/logger.go:50","msg":"app - Run - signal: interrupt","app-name":"go-service-template","app-version":"1.0.0"}
